ocfweb/settings.py

```diff
@@ -33,6 +34,7 @@
     'ocfweb.docs',
     'ocfweb.login',
     'ocfweb.main',
+    'ocfweb.error404',
```

Comment:
don't think we want an entire app just for this one page? maybe we could keep it under `main` or make a separate app for all errors?